<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LT-BLIOTTA" failures="0" tests="1" name="com.qa.SeleniumSessions.SikuliFlashObject" time="0.142" errors="1" timestamp="01 May 2020 14:56:05 GMT" skipped="0">
  <testcase classname="com.qa.SeleniumSessions.SikuliFlashObject" name="flashObjectsTest" time="0.142">
    <error type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at com.qa.SeleniumSessions.SikuliFlashObject.flashObjectsTest(SikuliFlashObject.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassCastException: class jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to class java.net.URLClassLoader (jdk.internal.loader.ClassLoaders$AppClassLoader and java.net.URLClassLoader are in module java.base of loader 'bootstrap')
at org.sikuli.script.RunTime.storeClassPath(RunTime.java:2515)
at org.sikuli.script.RunTime.isOnClasspath(RunTime.java:2562)
at org.sikuli.script.RunTime.isJarOnClasspath(RunTime.java:2586)
at org.sikuli.script.RunTime.libsExport(RunTime.java:935)
at org.sikuli.script.RunTime.init(RunTime.java:721)
at org.sikuli.script.RunTime.get(RunTime.java:298)
at org.sikuli.script.RunTime.get(RunTime.java:124)
at org.sikuli.script.RunTime.get(RunTime.java:323)
at org.sikuli.script.Region.<clinit>(Region.java:31)
... 26 more
]]>
    </error>
  </testcase> <!-- flashObjectsTest -->
</testsuite> <!-- com.qa.SeleniumSessions.SikuliFlashObject -->
